Pylint errors.

Pylint errors encountered during PR checks.

```
************* Module dftimewolf.lib.exporters.s3_to_gcs
dftimewolf/lib/exporters/s3_to_gcs.py:129:14: E1101: Instance of 'Resource' has no 'googleServiceAccounts' member (no-member)
************* Module tests.test_modules.thread_aware_modules
tests/test_modules/thread_aware_modules.py:140:2: W0221: Number of parameters was 0 in 'ThreadAwareModule.KeepThreadedContainersInState' and is now 1 in overridden 'Issue503Module.KeepThreadedContainersInState' method (arguments-differ)
```

--- a/dftimewolf/lib/collectors/grr_hosts.py
+++ b/dftimewolf/lib/collectors/grr_hosts.py
@@ -864,7 +864,7 @@ class GRRFlowCollector(GRRFlow):
     self.flow_id = str()
     self.host: containers.Host
 
-  # pylint: disable=arguments-differ
+  # pylint: disable=arguments-differ, arguments-renamed
   def SetUp(self,
             hostnames: str,
             flow_ids: str,
@@ -976,7 +976,7 @@ class GRRTimelineCollector(GRRFlow):
 
   # We're overriding the behavior of GRRFlow's SetUp function to include new
   # parameters.
-  # pylint: disable=arguments-differ,too-many-arguments
+  # pylint: disable=arguments-differ,too-many-arguments, arguments-renamed
   def SetUp(self,
             hostnames: str,
             root_path: str,

--- a/dftimewolf/lib/exporters/s3_to_gcs.py
+++ b/dftimewolf/lib/exporters/s3_to_gcs.py
@@ -127,7 +127,7 @@ class S3ToGCSCopy(module.ThreadAwareModule):
 
     See https://cloud.google.com/storage-transfer/docs/configure-access#sink"""
     request = self.dest_project.storagetransfer.GcstApi().\
-        googleServiceAccounts().get(projectId=self.dest_project_name)
+        googleServiceAccounts().get(projectId=self.dest_project_name)  # pylint: disable=no-member
     service_account = request.execute()['accountEmail']
 
     client = storage_client(project=self.dest_project_name)

--- a/tests/test_modules/thread_aware_modules.py
+++ b/tests/test_modules/thread_aware_modules.py
@@ -137,5 +137,5 @@ class Issue503Module(module.ThreadAwareModule):
   def PostProcess(self) -> None:
     pass
 
-  def KeepThreadedContainersInState(self) -> bool:
+  def KeepThreadedContainersInState(self) -> bool:  # pylint: disable=arguments-differ
     return False

